.github/workflows/integration-tests.yml

```diff
@@ -59,36 +59,54 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           wait-interval: 10
 
-  mock-integration:
-    needs: [await-python-checks]
-    runs-on: ubuntu-latest
-    services:
-      docker:
-        image: docker:24.0.5
-        options: --privileged
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
+  # cleanup:
+  #   runs-on: self-hosted
+  #   steps:
+  #     - name: Checkout deployment tooling
+  #       uses: actions/checkout@v4
+  #       with:
+  #         repository: simplyblock-io/simplyBlockDeploy
+  #         path: deploy
 
-      - name: Set up Docker Compose
-        uses: docker/setup-buildx-action@v2
+  #     - name: cleanup cluster
+  #       timeout-minutes: 15
+  #       run: |
+  #         cd deploy/bare-metal
+  #         cluster="${{ vars.ci_cluster }}"
+  #         cluster="${cluster:1}"
+  #         echo "cleaning up cluster $cluster"
+  #         KEY="~/.ssh/simplyblock-us-east-2.pem"
+  #         eval $(python3 inventory.py inventory/c${cluster}.yml)
 
-      - name: Build and start containers
-        run: |
-          docker compose -f docker-compose-dev.yml up --build -d
+  #         # restart all nodes
+  #         for node in $STORAGE_PRIVATE_IPS; do
+  #           echo "restart node $node"
+  #           ssh -i $KEY -o StrictHostKeyChecking=no root@$node "
+  #             (nohup bash -c 'sleep 1; reboot' >/dev/null 2>&1 &)
+  #             exit 0
+  #           "
+  #         done
 
-      - name: Run endpoint checks inside mgmt-server
-        run: |
-          docker compose -f docker-compose-dev.yml exec mgmt-server python manual_testing/check_endpoints.py
+  #         # remove partions on each nodes
+  #         for node in $STORAGE_PRIVATE_IPS; do
+  #           sleep 5
+  #           echo "Waiting for $node to come online..."
+  #           until ssh -i $KEY -o StrictHostKeyChecking=no -o ConnectTimeout=5 -o BatchMode=yes root@$node "echo 'Node is up'" 2>/dev/null; do
+  #             sleep 5
+  #           done
 
-      - name: Stop and clean up
-        if: always()
-        run: |
-          docker compose -f docker-compose-dev.yml logs
-          docker compose -f docker-compose-dev.yml down --volumes --remove-orphans
+  #           ssh -i $KEY -o StrictHostKeyChecking=no root@$node "
+  #             for disk in nvme0n1 nvme1n1 nvme2n1 nvme3n1; do
+  #               for part in 1 2; do
+  #                 echo "Removing partition \$part from disk \$disk"
+  #                 sudo parted /dev/\$disk rm \$part || true
+  #               done
+  #             done
+  #           "
+  #         done
 
   integration:
-    needs: [await-image, mock-integration]
+    needs: [await-image, await-python-checks]
```

Comment:
It seems you removed the mock-integration. Can we then also remove the `manual_testing/check_endpoints.py` script or is that used elsewhere?

Reply:
removed in [2b87f43](https://github.com/simplyblock-io/sbcli/pull/530/commits/2b87f432f6f508c6d0725ca00b10e089cc6dbcd0)